src/mplhep/plot.py

```diff
@@ -167,11 +172,91 @@ def histplot(
         else get_histogram_axes_title(hists[0].axes[0])
     )
 
-    # Cast to plottables
-    plottables = [
-        Plottable(h.values(), edges=final_bins, variances=h.variances()) for h in hists
-    ]
+    # Show under/overflow bins
+    ## check underflow/overflow bin exist
+
+    underflow, overflow = 0.0, 0.0
 
+    for h in hists:
+        if (
+            not isinstance(h, hist.hist.Hist)
```

Comment:
Also, can you check for `hist` type in the same way as you do for TH1? We don't need to make `hist` a dependency.

Reply:
Actually, perhaps it would be best to just check it functionally aka 
```
hasattr(h, 'values') and 'flow' in inspect.getfullargspec(h.values).args
```
which should work for both `hist` and `TH1` or anything random that will conform the `uhi` spec (https://uhi.readthedocs.io/en/latest/)